README.md

```diff
@@ -64,23 +64,36 @@ If you want to use this automation, read the instructions in [App Creation Scrip
 
 #### Register the service app (TaskTrackerWebApp-GroupClaims)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'TaskTrackerWebApp-GroupClaims' and select 'Web app / API' as the *Application Type*.
-1. For the *sign-on URL*, enter the base URL for the sample. By default, this sample uses `https://localhost:44322/`.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. For the App ID URI, replace the guid in the generated URI 'https://\<your_tenant_name\>/\<guid\>', with the name of your service, for example, 'https://\<your_tenant_name\>/TaskTrackerWebApp-GroupClaims' (replacing `<your_tenant_name>` with the name of your Azure AD tenant)
-1. From the **Settings** | **Reply URLs** page for your application, update the Reply URL for the application to be `https://localhost:44322/`
-1. From the Settings menu, choose **Keys** and add a new entry in the Password section:
-
-   - Type a key description (of instance `app secret`),
+1. In the  **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `TaskTrackerWebApp-GroupClaims`.
+   - Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+   - For the **Redirect URI (optional)**, select Web and enter the base URL for the sample. By default, this sample uses `https://localhost:44322/`.
+1. Click **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. Select the **Expose an API** section, and:

```

Comment:
@AllisonAm : I don't think that this sample exposes an API?
it's a Web app which calls Microsoft Graph?

Reply:
You are correct. You have to go to Expose an API to change the app ID URI as requested in the original version. I then got carried away based on the template, my apologies.

Question around this step- it was in the original guide, but if that URI is not being used, is there a reason we explicitly set it?

Reply:
No you are right, in the new app model, we should not even set it (it's a Web API thing)